test/spec/algoliasearch.helper/getQueryParameter.js

```diff
@@ -14,11 +13,9 @@ test('getQueryParameter', function(t) {
     ignorePlurals: true
   });
 
-  t.deepEqual(helper.getQueryParameter('facets'), ['facet1']);
-  t.equal(helper.getQueryParameter('minWordSizefor1Typo'), 8);
-  t.equal(helper.getQueryParameter('ignorePlurals'), true);
+  expect(helper.getQueryParameter('facets')).toEqual(['facet1']);
+  expect(helper.getQueryParameter('minWordSizefor1Typo')).toBe(8);
+  expect(helper.getQueryParameter('ignorePlurals')).toBe(true);
 
-  t.throws(bind(helper.getQueryParameter, helper, 'unknown'));
-
-  t.end();
+  expect(bind(helper.getQueryParameter, helper, 'unknown')).toThrow();
```

Comment:
We can add an inline snapshot.

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278627422